docs/docs/aztec/developer/wallet-providers/account-abstraction.md

```diff
@@ -19,7 +19,7 @@ The benefits of AA are multiple. We're not going to reiterate them all here, but
 
 ### Implementing at protocol vs application layer
 
-Account abstraction can be implemented at the application layer of a network using smart accounts and meta-transactions. The tx being sent to the network is still an Ethereum tx, but its payload is interpreted as a "transaction execution request" that is validated and run by the smart contract wallet. 
+Instead of implementing it at the protocol level as in Aztec, account abstraction can be implemented at the application layer of a network using smart accounts and meta-transactions. The tx being sent to the network is still an Ethereum tx, but its payload is interpreted as a "transaction execution request" that is validated and run by the smart contract wallet. 
```

Comment:
nit: tx => transaction may be worthwhile if it alleviates mental load for new readers